server/plugin/command.go

```diff
@@ -483,7 +500,9 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 
 	if f, ok := p.CommandHandlers[action]; ok {
 		message := f(c, args, parameters, info)
-		p.postCommandResponse(args, message)
+		if len(message) > 0 {
```

Comment:
Nit pick: It's idiomatic to use 
```suggestion
		if message != "" {
```
instead